src/main/java/org/jboss/ws/plugins/tools/WSContractDelegate.java

```diff
@@ -144,6 +144,9 @@ private void runConsumerOutOfProcess(WSContractConsumerParams params, String wsd
       {
          commandList.add(argLine);
```

Comment:
We also need to fix how java executable is called. Now it just calls "java" which is on path.
It should use system.getProperty("java.home")

Reply:
Done, thanks

Reply:
I have submitted PR 5 which has Alessio's changes and a fix so the tests run.